server/api/contract/extensions/erc1155/read/getOwned.ts

```diff
@@ -16,6 +16,18 @@ const querystringSchema = Type.Object({
     description: "Address of the wallet to get NFTs for",
     examples: ["0x1946267d81Fb8aDeeEa28e6B98bcD446c8248473"],
   }),
+  start: Type.Optional(
+    Type.Number({
+      description: "The start token id for paginated results. Defaults to 0.",
```

Comment:
Not necessarily the token id, more like the starting index

Reply:
Oh, I copied the whole text from the `getAll` endpoint. Let me update